src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionfurtherevidence/HandleEvidenceEventHandler.java

```diff
@@ -125,6 +132,18 @@ private SscsDocument buildSscsDocument(SscsCaseData sscsCaseData, ScannedDocumen
 
         DocumentLink url = scannedDocument.getValue().getUrl();
 
+        log.info("document link: " + url);
+
+        if (caseState != null && isIssueFurtherEvidenceToAllParties(sscsCaseData.getFurtherEvidenceAction())
+                && (caseState.equals(State.DORMANT_APPEAL_STATE)
```

Comment:
Have you missed out responseReceived state here?

Reply:
yes, It is missed. Should be there